podman_tutorial.md

```diff
@@ -0,0 +1,153 @@
+![PODMAN logo](https://cdn.rawgit.com/kubernetes-incubator/cri-o/master/logo/crio-logo.svg)
+
+# Basic Setup and Use of Podman
+Podman is a utility provided as part of the libpod library.  It can be used to create and maintain
+containers. The following tutorial will teach you how to set up podman and perform some basic
+commands with podman.
+
+## Install podman on Fedora
```

Comment:
I'd make "podman" "Podman" in all places except for actual command invocations.

Reply:
done